python/cudf/cudf/core/frame.py

```diff
@@ -21,7 +21,13 @@ class Frame(libcudfxx.Table):
     def _from_table(cls, table):
         return cls(table._data, index=table._index)
 
+    def as_column(self):
```

Comment:
Should we make this an internal method? Otherwise this will be exposed for Series / Index / etc.

Something like:
```python
@property
def _column(self):
    if self._num_columns > 1:
        raise RuntimeError
    return next(iter(self._data.values())
```